config/externalnamenottested.go

```diff
@@ -590,4 +590,115 @@ var ExternalNameNotTestedConfigs = map[string]config.ExternalName{
+	"aws_accessanalyzer_archive_rule": config.TemplatedStringAsIdentifier("", "{{ .parameters.analyzer_name }}/{{ .parameters.rule_name }}"),
+
+	// acmpca
+	//
+	// No import
+	// TODO: For now API is not normalized. While testing resource we can check the actual ID and normalize the API.
+	"aws_acmpca_permission": config.IdentifierFromProvider,
+	// aws_acmpca_policy can be imported using the resource_arn value
+	// Example: arn:aws:acm-pca:us-east-1:123456789012:certificate-authority/12345678-1234-1234-1234-123456789012
+	"aws_acmpca_policy": config.IdentifierFromProvider,
+
+	// appconfig
+	//
+	// AppConfig Extensions can be imported using their extension ID
+	"aws_appconfig_extension": config.IdentifierFromProvider,
+	// AppConfig Extension Associations can be imported using their extension association ID
+	"aws_appconfig_extension_association": config.IdentifierFromProvider,
+
+	// applicationinsights
+	//
+	// ApplicationInsights Applications can be imported using the resource_group_name
+	"aws_applicationinsights_application": config.ParameterAsIdentifier("resource_group_name"),
+
+	// apprunner
+	//
+	// App Runner Observability Configuration can be imported by using the arn
+	// Example: arn:aws:apprunner:us-east-1:1234567890:observabilityconfiguration/example/1/d75bc7ea55b71e724fe5c23452fe22a1
+	"aws_apprunner_observability_configuration": config.IdentifierFromProvider,
+	// App Runner VPC Ingress Connection can be imported by using the arn
+	// Example: arn:aws:apprunner:us-west-2:837424938642:vpcingressconnection/example/b379f86381d74825832c2e82080342fa
+	"aws_apprunner_vpc_ingress_connection": config.IdentifierFromProvider,
+
+	// appsync
+	//
+	// Appsync Types can be imported using the id (api-id:format:name)
+	"aws_appsync_type": config.IdentifierFromProvider,
+
+	// auditmanager
+	//
+	// Audit Manager Account Registration resources can be imported using the id
+	"aws_auditmanager_account_registration": config.IdentifierFromProvider,
+	// Audit Manager Assessments can be imported using the assessment id (abc123-de45)
+	"aws_auditmanager_assessment": config.IdentifierFromProvider,
+	// Audit Manager Assessment Reports can be imported using the assessment report id (abc123-de45)
+	"aws_auditmanager_assessment_report": config.IdentifierFromProvider,
+	// An Audit Manager Control can be imported using the id (abc123-de45)
+	"aws_auditmanager_control": config.IdentifierFromProvider,
+	// Audit Manager Framework can be imported using the framework id (abc123-de45)
+	"aws_auditmanager_framework": config.IdentifierFromProvider,
+
+	// ce
+	//
+	// aws_ce_anomaly_monitor can be imported using the id
+	"aws_ce_anomaly_monitor": config.IdentifierFromProvider,
+	// aws_ce_anomaly_subscription can be imported using the id
+	"aws_ce_anomaly_subscription": config.IdentifierFromProvider,
+	// aws_ce_cost_allocation_tag can be imported using the id
+	"aws_ce_cost_allocation_tag": config.IdentifierFromProvider,
```

Comment:
I would suggest treating `tag_key` as the naming argument and the ID for this resource here and validating this during the tests:
```suggestion
	"aws_ce_cost_allocation_tag": config.ParameterAsIdentifier("tag_key"),
```